calcom/cal.com
# perf: getSchedule - Running queries in parallel

## What does this PR do?

This breaks the queries needed to fulfill `getEventType` into separate async calls not relying on Prisma when calling `getSchedule`. On the dev DB, this is proving to be a consistent improvement of around 300ms.

## Type of change




## How should this be tested?



- Are there environment variables that should be set? **No**
- What are the minimal test data to have? **basic event types, dynamic event types, team event types**
- What is expected (happy path) to have (input and output)? **The getSchedule call functions exactly the same as before**

## Mandatory Tasks
